.buildkite/pipeline.full.yml

```diff
@@ -3,21 +3,25 @@ agents:
 
 steps:
   - label: 'Append Unity 2020 Pipeline'
+    depends_on: "build-unitypackage"
```

Comment:
These are needs so that we can trigger a full build without waiting for the notifier package to build first.